test/e2e/generator/main.go

```diff
@@ -86,21 +82,20 @@ func (cli *CLI) generate(dir string, groups int, opts Options) error {
 		return err
 	}
 	if groups <= 0 {
-		for i, manifest := range manifests {
-			err = manifest.Save(filepath.Join(dir, fmt.Sprintf("gen-%04d.toml", i)))
-			if err != nil {
-				return err
-			}
+		e2e.SortManifests(manifests)
+
+		if err := e2e.WriteManifests(filepath.Join(dir, "gen"), manifests); err != nil {
+			return err
 		}
 	} else {
```

Comment:
Optional suggestion: `return nil` at the end of this block (since that's what happens anyway) and outdent the "interesting" else case.

Reply:
I'm not sure that the other one is "more interesting" (certainly more complex,) and they seem both like peers, and I think changing the indent would be more confusing? I made it a switch statement :shrug: